src/main/resources/static/api.yaml

```diff
@@ -2602,6 +2662,30 @@ components:
         - username
         - password
         - gender
+    RecoverCmIdRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Raj Sharma
+        gender:
+          $ref: '#/components/schemas/Gender'
+        phoneNumber:
```

Comment:
Phone number should also be specified as "Identifier" :  { "identifierType" : "MOBILE", "identifierValue" : "phone number" }

Reply:
added "Mobile" to verified Identifiers